src/createServer.js

```diff
@@ -1,8 +1,52 @@
+/* eslint-disable no-console */
 'use strict';
 
+const http = require('http');
+const fs = require('fs');
+const url = require('url');
+
 function createServer() {
-  /* Write your code here */
-  // Return instance of http.Server class
+  const server = http.createServer((req, res) => {
+    const { pathname } = new url.URL(req.url, `http://${req.headers.host}`);
+    const filename = pathname.replace('/file/', '') || 'index.html';
+
+    res.setHeader('Content-Type', 'text/plain');
+
+    if (pathname.includes('//')) {
+      res.statusCode = 404;
```

Comment:
```suggestion
      res.statusCode = 400;
```

400 error stands for 'Bad request'